Ptt/Common/BoardSearch/BoardSearchViewController.swift

```diff
@@ -72,16 +72,21 @@ final class BoardSearchViewController: UITableViewController {
 
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         tableView.deselectRow(at: indexPath, animated: true)
-#if DEVELOP // Disable editing, for now
+        // Based on "Ptt 官方 app 測試小組" unofficial survey in 2023-08-14,
+        // voting for expected behavior is
+        // 8:0 for showBoardView : addBoardToFavorite
```

Comment:
IMO, open a board is the expected behavior..... I probably had brain fog at that time :(   
anyway ... do we need these comments?